addons/binding/org.openhab.binding.yeelight/about.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+		"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
+	<title>About</title>
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+
+<p>March 30, 2017</p>
+<h3>License</h3>
+
+<p>
+	The openHAB community makes available all content in this plug-in (&quot;Content&quot;). Unless otherwise
+	indicated below, the Content is provided to you under the terms and conditions of the
+	Eclipse Public License Version 1.0 (&quot;EPL&quot;). A copy of the EPL is available
+	at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
+	For purposes of the EPL, &quot;Program&quot; will mean the Content.
+</p>
+
+<p>
+	If you did not receive this Content directly from the openHAB community, the Content is
+	being redistributed by another party (&quot;Redistributor&quot;) and different terms and conditions may
+	apply to your use of any object code in the Content. Check the Redistributor's license that was
+	provided with the Content. If no such license exists, contact the Redistributor. Unless otherwise
+	indicated below, the terms and conditions of the EPL still apply to any source code in the Content
+	and such source code may be obtained at <a href="http://www.openhab.org/">openhab.org</a>.
+</p>
+
+
+<h3>Third Party Content</h3>
+<p>The Content includes items that have been sourced from third parties as set out below. If you
+	did not receive this Content directly from the Eclipse Foundation, the following is provided
+	for informational purposes only, and you should look to the Redistributor's license for
+	terms and conditions of use.</p>
+<p>
+	<strong>yeelight_lib Library</strong> <br/><br/>
+	This bundle includes yeelight library for sending control command to Yeelight devices. The sources
+	can be found at
+	<a href="https://github.com/coasterli/YeelightLib">GitHub</a> and are made available under the
```

Comment:
Is this a stable location, should someone fork it?

Reply:
For now it was stable (over a year I guess) but for the future it might be a good idea to fork it, so it cannot simply disappear. Who should to do the fork, also shell I change the link then or only in case that repo gets unavailable?

Reply:
I could create the fork under the openhab org, then we could be sure that it won't disappear.

Reply:
Hm, what is worse, I just saw https://github.com/coasterli/YeelightLib/issues/1. This actually means that the code that is included here in the binding is NOT available as open source anywhere - we thus should not include it. It would be really helpful if @coasterli would respond to those issues...

Reply:
I guess he won't in the near future, although I could try to contact him again via the Yeelight forum. For now, we should assume that he won't response. Does that mean this cannot be included into the addon repo of openHAB? Is there a chance to get it into the Eclipse SmartHome Market?

Reply:
I just had a look into the jar file: The good news is that it actually includes the sources!
As the official lib is actually dead, I wonder if we should directly create a `src/3rdparty/java` source folder and put the source code directly in the binding (so that it can be easily adapted and maintained there as well).
@martinvw, wdyt about that?

Reply:
Nice observation!
However, can this decision be postponed into an issue and implemented in a new PR later?

Reply:
We should in any case update the jar, since it not only contains source files, but also the classes twice as well as a gson.jar. All of that would have to be removed. So it might be actually easier to simply take the source out.

Reply:
After looking at the code, I am not so sure I would want the sources in our repo. So I'd be fine if you just remove the stuff that I mentioned and rather add a second jar that includes the source file (so we have them stored somewhere), but isn't added in the build.properties.

Reply:
I am not sure whether I can do that in time. There are also other issues when actually using the binding that I listed below and need addressing. I will just continue working on this preferring the quick lib way. Maybe it works out before the deadline, maybe not.

If there is somebody else willing to help with some of the tasks like handling the lib feel free to create PRs to my repo, I will make sure to merge them in time.

Reply:
@kaikreuzer I agree with your final conclusion :-)

Sorry I'm and was a little bit busy with some last minute GDPR things for my own company and some other volunteering work I do ;-)

Reply:
> If there is somebody else willing to help with some of the tasks like handling the lib

I'll create a PR for this asap.

Reply:
Oh - you just pushed it - thanks :-)

Reply:
If you test your version, you will notice that it won't work; you have kept the wrong class files; there must not be "target/classes" folder, in the root of the jar must be directly "com".

Reply:
I'll nonetheless create a PR to clean up these jars.

Reply:
Yes, noticed this can be easily done with a zip manager. I will push a new version with "com".

Reply:
I guess there are still some problems with my modified lib, since Jenkins fails.

Reply:
You didn't merge my PR yet, did you?

Reply:
Sorry, probably overlooked it. Merged now, many thanks!